fabric-enchantments-v1/src/main/java/net/fabricmc/fabric/impl/enchantment/EnchantmentTargetRegistryImpl.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.enchantment;
+
+import net.fabricmc.fabric.api.enchantment.EnchantmentTargetRegistry;
+import net.fabricmc.fabric.api.enchantment.FabricEnchantmentTarget;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class EnchantmentTargetRegistryImpl implements EnchantmentTargetRegistry {
+	public static final EnchantmentTargetRegistryImpl INSTANCE = new EnchantmentTargetRegistryImpl();
+
+    public static final Set<FabricEnchantmentTarget> ENCHANTMENT_TARGETS = new HashSet<>();
```

Comment:
mind changing this field to be an instance field?

Reply:
Done 🙂